CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 1.5.1
 Release built: _not published yet_
```

Comment:
Can we inform about database changes around schemas?

Reply:
Added few words.